ateam_common/include/ateam_common/status.hpp

```diff
@@ -137,6 +127,7 @@ StatusOr<T> Failure(std::string && s)
       if (!__status_or_lhs) { \
         std::stringstream s; \
         s << text << "\n" << __status_or_lhs.error() << std::endl; \
+        std::cout << s.str(); \
```

Comment:
Is this a leftover debug printout?
If not, should it print to `std::cerr` instead?

Reply:
Will change to std::cerr